lib/Runtime/Library/JavascriptProxy.cpp

```diff
@@ -2178,13 +2179,19 @@ namespace Js
                 calleeInfo.Flags = (CallFlags)(calleeInfo.Flags | CallFlags_ExtraArg | CallFlags_NewTarget);
             }
 
-            for (uint argCount = 0; argCount < args.Info.Count; argCount++)
+            for (ushort argCount = 0; argCount < (ushort)args.Info.Count; argCount++)
```

Comment:
>args.Info.Count [](start = 57, length = 15)

is it safe to cast args.Info.Count to ushort? If so, can we make args.Info.Count have the correctly-sized type?

Reply:
Further up around https://github.com/Microsoft/ChakraCore/pull/5272/files#diff-771647fb11e03690664371970f34eeaaR2126 we check that the value fits in a ushort and throw an error if it doesn't. We do allow for larger argument counts in some circumstances though, so `args.Info.Count` is a uint.